src/overlays/actors/ovl_En_Invisible_Ruppe/z_en_invisible_ruppe.c

```diff
@@ -93,30 +93,32 @@ void EnInvisibleRuppe_Init(Actor* thisx, GlobalContext* globalCtx) {
     EnInvisibleRuppe* this = THIS;
     s32 pad;
 
-    this->unk_190 = (this->actor.params & 0x1FC) >> 2;
+    this->unk_190 = INVISIBLERUPPE_GET_7F0(this);
```

Comment:
7FC, I think?

Reply:
how did you get that number?

Reply:
1FC, then? Where does 7F0 come from?

Reply:
I looked at other examples, like in [z_en_dno.h](https://github.com/zeldaret/mm/blob/master/src/overlays/actors/ovl_En_Dno/z_en_dno.h#L13). `3F80` seems to come from `0x7F << 7`, so I did ` 0x1FC << 2`.

Sorry if my question came across as accusatory. I'm genuinely curious where these numbers come from.

Reply:
No, that's fine, you're right that I made a typo (based on what I assumed was your typo lol).

The difference here is that the & happens before the shift here and after on that one. So it's getting the 0x1FC part and shifting after.

Reply:
Ah yeah, okay. I'll just rename it to 1FC.